resources/views/layouts/app.blade.php

```diff
@@ -5,16 +5,23 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <link rel="stylesheet" href="{{ asset('/static/css/app.css') }}">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+        integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <script defer src="https://unpkg.com/alpinejs@3.x.x/dist/cdn.min.js"></script>
     <title>Meetup app</title>
+    <style>
+        [x-cloak] {
+            display: none !important;
+        }
+    </style>
```

Comment:
This will be useless if u import `alpine.js` as a package.

Reply:
Although I installed this package, this line is still neccessary. I added this line to app.css, where are tailwind classes, so i don't know if this is a good choice.